content/en/blog/_posts/2021-01-28-PSP-deprecation.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: blog
+title: 'What deprecation of Pod Security Policies means'
+date: 2021-01-28
+slug: psp-deprecation
+---
+
+**Authors:** Matt Broberg (Red Hat), Kaslin Fields (Google), Chris Short (Red Hat)
+
```

Comment:
```suggestion
_The following is a brief overview of upcoming changes to Kubernetes. Bookmark and [keep an eye on our release site](https://www.kubernetes.dev/resources/release/) to see the latest news and other important updates._
```

Reply:
^ This is intended to reinforce the scope from the beginning of the article. If there's a better place to link them, let's do it here. cc @justaugustus @divya-mohan0209

Reply:
@mbbroberg : I agree with this reinforcement of scope at the very start of the blog. It makes total sense to lay it out & convey that this is not a technical deep-dive and shouldn't be treated as one.